scene/2d/animated_sprite.cpp

```diff
@@ -481,6 +481,16 @@ int AnimatedSprite::get_frame() const {
 	return frame;
 }
 
+void AnimatedSprite::set_speed_factor(float p_speed_factor) {
+
+	speed_factor = MAX(p_speed_factor, 0.f);
```

Comment:
It's a matter of style, but since we use `0.0f` most of the time, would be good to stick to it here too.